src/context.rs

```diff
@@ -191,7 +188,8 @@ mod alloc_only {
         /// ctx.seeded_randomize(&seed);
         /// # }
         /// ```
-        #[allow(unused_mut)]    // Unused when `rand-std` is not enabled.
+        #[allow(unused_mut)]             // Unused when `rand-std` is not enabled.
+        #[allow(clippy::let_and_return)] // Triggers when `rand-std` is not enabled.
```

Comment:
Perhaps `cfg_attr` to limit the scope? Not a big deal though.

Reply:
Nice trick! I had to think about your suggestion for a bit then I worked it out
```rust
    #[cfg_attr(not(feature = "rand-std"), allow(clippy::let_and_return, unused_mut))]
```
For the second time today; thanks for the lesson :)